app/client/src/pages/aboutus.tsx

```diff
@@ -1,52 +1,112 @@
 import * as React from 'react';
+import Image from 'next/image';
 import { Grid, Typography } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         width: '100%',
         height: '100%',
+        paddingLeft: theme.spacing(1),
+        textAlign: 'center',
     },
-    paper: {
-        padding: theme.spacing(3),
+    section: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        gap: '1rem',
+        textAlign: 'center',
+    },
+    paragraph: {
+        fontSize: 'large'
     },
 }));
 
-const AboutUs = () => {
+export default function AboutUs() {
     const classes = useStyles();
+
     return (
-        <Grid className={classes.root} container direction='column' justify='center' alignItems='center' spacing={3}>
-            <Grid item>
-                <Typography variant='h1'>About Us</Typography>
+        <Grid container className={classes.root} justify='center' alignItems='center' spacing={2}>
+            <Grid container item xs={12} direction='column' alignItems='center' spacing={1}>
+                <Grid item xs={4} sm={3} md={2}>
+                    <Image
+                        src='/static/prytaneum_logo.svg' 
+                        width={1107}
+                        height={1108}
+                        objectFit='contain'
+                    />
+                </Grid>
+                <Grid item>
+                    <Typography variant='body1' className={classes.paragraph}>
```

Comment:
Rather than using `classes.paragraph` to dictate text size, try using a different [variant](https://mui.com/api/typography/#props) (you'll need to scroll down to the bottom of the prop list).